packages/nimble-components/src/patterns/dropdown/styles.ts

```diff
@@ -110,6 +111,14 @@ export const styles = css`
         border-color: rgba(${borderRgbPartialColor}, 0.1);
     }
 
+    :host([error-visible]) .control,
+    :host([error-visible][open]) .control,
+    :host([error-visible][disabled]) .control,
+    :host([error-visible]):before,
+    :host([error-visible]):after {
+        border-bottom-color: ${failColor};
+    }
```

Comment:
Please organize the selectors so they are grouped by the selector target (the host:before and host:after selectors should move to the right spots): https://github.com/ni/nimble/blob/main/packages/nimble-components/docs/css-guidelines.md#group-selectors-by-target-and-document-order

Reply:
Done